docs/REST API/rest-api.md

```diff
@@ -767,6 +814,53 @@ See [Add certificate authority](#add-certificate-authority-ca) above for the str
 }
 ```
 
+### List certificates
+
+List certificate added to Fleet with [Add certificate](#add-certificate).
+
+`GET /api/v1/fleet/certificates`
+
+#### Parameters
+
+| Name      | Type    | In   | Description                                                    |
+| ----------| ------- | ---- | -------------------------------------------------------------- |
+| team      | string  | query | _Available in Fleet Premium_. The team ID to filter profiles. |
+| page      | integer | query | Page number of the results to fetch.                          |
+| per_page  | integer | query | Results per page.                                             |
+
+#### Example
+
+`GET /api/v1/fleet/certificates`
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "certificates": [
+    {
+      "id": 1,
+      "name": "wifi-certificate",
+      "certificate_authority_id": "1",
+      "certificate_authority_name": "PRODUCTION_SCEP_SERVER",
+      "created_at": "2025-11-04T00:00:00Z",
+    },
+    {
+      "id": 2,
+      "name": "vpn-certificate",
+      "certificate_authority_id": "1",
+      "certificate_authority_name": "PRODUCTION_SCEP_SERVER",
+      "created_at": "2025-11-04T00:00:00Z",
+    }  
+  ],
+  "meta": {
+    "has_next_results": false,
+    "has_previous_results": false
+  }
+}
+```
+
```

Comment:
Is this used anywhere in https://github.com/fleetdm/fleet/issues/35198? If not can we cut if for scope?

Reply:
Actually we probably need something like this for GitOps, but more likely another `/spec` endpoint.